stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift

```diff
@@ -136,6 +136,50 @@ public struct FindTest {
   }
 }
 
+public struct CollectionTypeTest {
+  public let expected: [MinimalEquatableValue]
+  public let a: [MinimalEquatableValue]
+  public let b: [MinimalEquatableValue]
+  public let loc: SourceLoc
+
+  public init(
+    expected: [Int], a: [Int], b: [Int],
+    file: String = #file, line: UInt = #line
+  ) {
+    self.expected = expected.enumerated().map {
+      return MinimalEquatableValue($1, identity: $0) 
+    }
+    self.a = a.enumerated().map {
+      return MinimalEquatableValue($1, identity: $0) 
+    }
+    self.b = b.enumerated().map {
+      return MinimalEquatableValue($1, identity: $0) 
+    }
+    self.loc = SourceLoc(file, line, comment: "test data")
+  }
+}
+
+public struct StrictSupersetTest {
```

Comment:
Similarly, I think this can be called `CollectionPredicateTest`, this way we will be able to reuse this type for many datasets.